third_party/cpuinfo.BUILD

```diff
@@ -0,0 +1,62 @@
+# UATC LLC (c) 2019
```

Comment:
@werkt can you please either remove copyright headers from .BUILD files or expand it to full BSD-style headers?

Reply:
Done. I missed this in my conversion, after looking for it and being confused that I hadn't found any of our copyrights.